extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -89,7 +89,7 @@ config.run('amp-image-slider', function() {
         // layoutCallback is called
```

Comment:
Suggest monitoring master red for flakes here and see if there are any platforms (e.g. Chrome on Windows) that need to be skipped due to the lower timeout.